service/hub/server/config.go

```diff
@@ -8,10 +8,23 @@ import (
 	"os"
 	"strings"
 
+	"github.com/cesanta/glog"
 	"github.com/docker/libtrust"
+	"github.com/labring/sealos/pkg/client-go/kubernetes"
 	yaml "gopkg.in/yaml.v2"
+	ctrl "sigs.k8s.io/controller-runtime"
 )
 
+var K8sClient kubernetes.Client
+
+func init() {
+	var err error
+	K8sClient, err = kubernetes.NewKubernetesClientByConfig(ctrl.GetConfigOrDie())
```

Comment:
Can we init k8s client at first anonymous call instead of the time just server starts?